dataset_generator/fuzzing-script.py

```diff
@@ -34,46 +46,34 @@ def generate_expr_exc(exclude, depth=0):
     op = random.choice(OP)
     left = generate_expr_exc(exclude, depth + 1)
     right = generate_expr_exc(exclude, depth + 1)
-    return f"({op} {left} {right})"
-
-def tokenize(expr: str):
-    expr = expr.replace('(', ' ( ').replace(')', ' ) ')
-    return expr.split()
-
-def eval_tokens(tokens):
-    def helper():
-        token = tokens.pop(0)
-        if token == '(':
-            op = tokens.pop(0)
-            left = helper()
-            right = helper()
-            tokens.pop(0)
-            if op == '+': return left + right
-            if op == '-': return left - right
-            if op == '*': return left * right
-            if op == '/':
-                if right == 0:
-                    return 0
-                return int(left / right)
-        elif token.isdigit():
-            return int(token)
-        else:
-            return var_values.get(token, 0)
-    return helper()
-
-#function for later use to evaluate expressions
-def expr_eval(expr: str):
-    tokens = tokenize(expr)
-    return eval_tokens(tokens)
+    return op, left, right
+
+def eval_expr_tree(tree):
```

Comment:
If you really need to evaluate an expression you can call out to the interpreter, but I really don't think that should be necessary.  In any case, it isn't needed for this PR, so please remove this part.